runelite-client/src/main/java/net/runelite/client/plugins/mining/MiningPlugin.java

```diff
@@ -151,25 +147,10 @@ public void onAnimationChanged(final AnimationChanged event)
 			return;
 		}
 
-		int animId = local.getAnimation();
-		if (animId == AnimationID.DENSE_ESSENCE_CHIPPING)
```

Comment:
Did you check this handles dense essence ok?

Reply:
i did check it but i was just running around to all the rocks in the same session, when i just logged in next to it and tried to start a session, you're right, it didn't work correctly.  I put the [dense essence session start](https://github.com/runelite/runelite/compare/83f8076494050f8e9a4739bf2f4ed1692a8c4c54..0aff6bfd6e7efeb632351a3f9c145eec9b017a41) back now.